elements/nuxeo-results/nuxeo-results.js

```diff
@@ -191,7 +191,7 @@ Polymer({
       <div class="resultActions" hidden$="[[hideContentViewActions]]">
         <div class="commonActions">
           <span class="resultsCount" hidden$="[[!_showResultsCount(nxProvider, resultsCount)]]"
-            >[[i18n('results.heading.count', resultsCount)]]</span
+            >[[_resultsCountLabel]]</span
```

Comment:
can we fix this weird indentention?
```suggestion
            >[[_resultsCountLabel]]</span>
```